Source/SteveSloth/Private/MyEnemyAIDeadState.cpp

```diff
@@ -0,0 +1,34 @@
+/****************************************************************************************
+ * Copyright:
+ * Name: Guy Lapensee
+ * Script: ItemBaseClass
+ * Date: April 29, 2024
+ * Description: Enemy AI dead state for enemy base class
+ * TODO:
+ * Known Bugs:
+ ****************************************************************************************/
+
+
+#include "MyEnemyAIDeadState.h"
+
+MyEnemyAIDeadState::MyEnemyAIDeadState()
```

Comment:
In the constructor is where you should be calling your functions. This constructor is called when the StateMachine first calls this class.
This is the starting point of where the code needs to start and function.